docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -18,56 +18,105 @@ authenticate to external services.
   Reference](./saml-reference.mdx) before proceeding.
 - User with permission to create service provider resource. The preset `editor` role has this permission.
 - SAML application (also known as a SAML service provider or SP) for testing. For this guide, we'll be using
-[samltest.id](https://samltest.id/) as a service provider for Teleport's SAML IdP.
+[RSA Simple Test Service Provider](https://sptest.iamshowcase.com/) (iamshowcase app) - a free test service that
+let's us test Teleport SAML IdP. The test service has a protected page, which can be accessed only after a
+user is federated to the site with a valid SAML assertion flow.
```

Comment:
Should "federated" be "authenticated" here?

Reply:
Federated is the correct term here. 

In SAML, once a user is successfully authenticated in the IdP, they are federated into the service provider. In this case, user is federated to the "Simple Test Service Provider".